colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,370 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "7Ix_ciolJICe"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basic\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\")\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "u60Orwg5sYuI"
+      },
+      "source": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Create a new Artifact version"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "When you need add a new file, you can use the [`artifact.add_file`](https://docs.wandb.ai/ref/python/artifact#add_file) method."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(\"my_first_artifact:latest\") # selects the artifact you're adding the file to\n",
+        "artifact.add_file(local_path=\"/content/sample_data/california_housing_test.csv\", name=\"new_file\")\n",
+        "run.log_artifact(artifact)\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "This adds a new .csv file called `new_file` to the `my_first_artifact` Artifact. \n",
+        "\n",
+        "If you edit a file, you'll need to go through a similar process:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# sorts the .csv file\n",
+        "import pandas\n",
+        "csvData = pandas.read_csv(\"/content/sample_data/california_housing_test.csv\") \n",
+        "csvData.sort_values(csvData.columns[6],  \n",
+        "                    axis=0, \n",
+        "                    inplace=True)\n",
+        "csvData.to_csv(\"/content/sample_data/california_housing_test.csv\") # overwrites file with the sorted data\n",
+        "# adds the new file to the artifact\n",
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(\"my_first_artifact:latest\")\n",
+        "artifact.add_file(local_path=\"/content/sample_data/california_housing_test.csv\", name=\"sorted_file\")\n",
+        "run.log_artifact(artifact, aliases= [\"sorted\"]) # logs the new artifact version, overwriting the old one.\n",
+        "run.finish()\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Now the sorted file will be logged in `my_first_artifact`. Any changes you log to an artifact will overwrite any older version. \n",
```

Comment:
hmm the wording of overwriting old versions may be confusing to users (line 228 and 236) as we don't really overwrite the old version, we create a new version instead. Overwriting to me implies it replaces the previous.